view/frontend/web/js/view/payment/method-renderer/adyen-oneclick-method.js

```diff
@@ -314,18 +318,7 @@ define(
                                 // render component
```

Comment:
is getCcData on Line 288 still used?

Reply:
don't think so
I think we can remove it